### ISSUE TITLE
VSCode Insiders: Command Line Extension Management doesn't work

### ISSUE THREAD

**Issue by BillDenton** [Microsoft/vscode#22905]
- VSCode Version: Code - Insiders 1.11.0-insider (207686d256dafb78b247357ad8b96a95218a22a5, 2017-03-17T08:14:15.805Z)
- OS Version: Windows_NT ia32 6.1.7601
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|language-x86-64-assembly|13xforever|2.2.9|;

---

Steps to Reproduce:

1.  Open cmd in the Insiders directory "C:\Program Files (x86)\Microsoft VS Code Insiders"
2.  Type `"Code - Insiders.exe" --list-extensions`. This doesn't list any extension, but starts VSCode Insiders.
3. Typing `code --list-extensions` in the same command prompt produces a list of installed extensions in release version of VSCode.
3. Typing `"Code - Insiders.exe" --install-extension ms-vscode.cpptools` just started VSCode Insiders and doesn't install the extension.
4. After installing ms-vscode.cpptools inside VSCode Insiders, typing `"Code - Insiders.exe" --uninstall-extension ms-vscode.cpptools` just starts VSCode Insider and doesn't uninstall the extension.

**Comment by Tyriar** [comment]
The reason `code` works is because it points to `code.cmd` which runs the CLI, the executable cannot unfortunately. You will want to stick with `code` or `code-insiders` on stable, the latter of which is probably the answer to your problem :smiley:

**Comment by BillDenton** [comment]
Thanks. I have confirmed using "code-insiders.cmd" from the VS Code Insiders "bin" directory works as expected.
Normally I would stick with the release "code" build, but I was attempting to switch to the insider build to avoid a defect https://github.com/Microsoft/vscode/issues/12448. This particular defect is causing me a lot of irritation. To quickly transfer my extensions to the insider build, I listed them on the release build and was going to make a short bat file to install them into the insider build. I've now done this manually. Once the release build includes the fix for https://github.com/Microsoft/vscode/issues/12448 I'll switch back.
